test/common.js

```diff
@@ -53,4 +53,22 @@ describe("common", function() {
     });
 });
 
+describe("hash_password function",function(){
+    it('should has a password and return the password', async function(){
+        const password = 'testingPassword@987654';
+        const hash = await common.hash_password(password);
+        assert.ok(hash);
```

Comment:
you could use common.check_password or something to see if the generated password is valid

Reply:
umm, common.check_password is used to check if it matches with the hashed password of existing user. 
The hash_password function handles the case to check if password is strong, do you think I should create another function which could serve the purpose ? 
We only care if the strength > 0 .....

Reply:
I would change the `common.check_password` to receive two password hashes, instead of receiving the user object.